spec/acceptance/grafana_folder_spec.rb

```diff
@@ -21,6 +21,7 @@ class { 'grafana':
   context 'create folder resource' do
     it 'creates the folder' do
       pp = <<-EOS
+      include grafana::validator
```

Comment:
thinking about it, it's required now to include the classe because the other types require it? In that case this a a backwards incompatible change?

Reply:
The only requires are autorequire which only require something if in catalog so if someone doesn't include the new class, nothing changes. This change is essentially opt-in.